apps/hash-graph/tests/integration/postgres/links.rs

```diff
@@ -2,12 +2,12 @@ use graph::knowledge::{EntityLinkOrder, EntityProperties};
 use graph_test_data::{data_type, entity, entity_type, property_type};
 use type_system::uri::{BaseUri, VersionedUri};
 
-use crate::postgres::DatabaseTestWrapper;
+use crate::DatabaseTestWrapper;
 
 #[tokio::test]
 async fn insert() {
-    let person_a = serde_json::from_str(entity::PERSON_A_V1).expect("could not parse entity");
-    let person_b = serde_json::from_str(entity::PERSON_B_V1).expect("could not parse entity");
+    let alice = serde_json::from_str(entity::PERSON_A_V1).expect("could not parse entity");
```

Comment:
Should we rename `PERSON_A_V1` to `PERSON_ALICE_V1` then? I think `alice` and `bob` in code is less likely to get mixed up compared to `person_a` and `person_b` so this is probably a good change, I just don't want us randomly giving variables human names if the thing inside it doesn't relate.

So perhaps us renaming it to `PERSON_ALICE_V1` will also help us try and make sure the contents of the JSON stay as "alice"

Reply:
[Rename person entity file names](https://github.com/hashintel/hash/pull/2073/commits/65ddd8e02d8c00addfc6a0f1d095cd4f5a428ea9)

Reply:
Can you also rename the variable names in the `test_data` files? They're still `entity::PERSON_A_V1`

Reply:
Oh, I missed that, thanks! e61b7c84ee13a679540022e084747eb172b7b672